### DISCUSSION TITLE
Website Feedback

### DISCUSSION THREAD

**Discussion by egekorkan** [eclipse-thingweb/thingweb#8] [General]
If you have feedback about the redesign of our homepage, feel free to leave any comments here.

**Comment by Citrullin** [comment]
You should bring up the logos into the viewpoint. Big brand names build some trust.

**Reply by egekorkan** [reply]
Thanks @Citrullin for the feedback! Do you mean that they should be more prominently placed (more towards the top) or should be made bigger? Alternatively, we can make sure that they are all displayed all the time.

**Reply by Citrullin** [reply]
@egekorkan More prominently at the top. It should be visible on all devices when they open the page. Probably even helpful to add links to pages where the companies write about WoT. Blog posts etc.

**Reply by egekorkan** [reply]
@Citrullin we had a chat about this in today's call. We will put two alternatives in a PR where they are placed in the first and second parts of the website.

**Reply by JKRhb** [reply]
@SergioCasCeb has created this PR in the meantime: https://github.com/eclipse-thingweb/website/pull/102

**Comment by benfrancis** [comment]
Congratulations on your new website, it looks great!

May I suggest you make a clearer distinction between the Eclipse Thingweb community and the W3C WoT Community Group. E.g. most of the community links in the footer refer to the W3C Community Group, which implies they are one and the same.

The World Wide Web Consortium and Eclipse Foundation are two separate organisations and ThingWeb is only one implementation of the W3C WoT standards, which is not really clear from the website.

**Reply by egekorkan** [reply]
Good points. We have built it from the pov of an IoT developer and didn't see much of a difference from that pov. Footnote can be easily fixed. For the differentiation to w3c, any core/body of the Website that feels weird?

**Reply by benfrancis** [reply]
Not that it's a big deal at all but just as an example the "130+ Discord Members" presumably counts everyone on the Web of Things Community Discord server, which likely includes people who have never used node-wot etc.

More importantly, I think it would be useful to have an introduction or about page which explains what the Web of Things is, and that Eclipse Thingweb is an open source implementation of W3C WoT standards. The fact that there are multiple interoperable implementations of those standards is a key strength and something I think an IoT developer would want to know.

**Reply by egekorkan** [reply]
@benfrancis we have talked about this in today's Thingweb call. We are going to do the following:
- Join the Thingweb Community -> Join the Thingweb and WoT Community
- Below that, add a new text saying something like: Thingweb is part of the wide W3C WoT community and ecosystem
- 130+ Discord members text -> 130+ Members in the WoT CG Discord
- Explanation about what is wot will in a documentation subpage
- Footer with Community Header -> Entire WoT Community (or something similar)
- Social Media channels not owned by Eclipse Thingweb will get a superscript with an explanation below. They will be also separated with a horizontal line

Let us know if you have comments :)
